samples/matter/closure/src/app_task.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "app_task.h"
+
+#include "app/matter_init.h"
+#include "app/task_executor.h"
+#include "board/board.h"
+#include "closure_control_endpoint.h"
+#include "closure_manager.h"
+#include "lib/core/CHIPError.h"
+#include "lib/support/CodeUtils.h"
+
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app/data-model/Nullable.h>
+#include <app/util/attribute-storage.h>
+#include <setup_payload/OnboardingCodesUtil.h>
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+using namespace ::chip;
+using namespace ::chip::app;
+using namespace ::chip::DeviceLayer;
+using namespace chip::app::Clusters::Descriptor;
+
+static const struct pwm_dt_spec sPhysicalIndicatorPwmDevice = PWM_DT_SPEC_GET(DT_ALIAS(pwm_led1));
```

Comment:
I can see that only the ClosureManager uses this PWMDevice, so why do you need to add it to AppTask, and then pass to mClosureManager? Isn't it better initalize it directly in ClosureManager's constructor?